src/utils/validation.js

```diff
@@ -0,0 +1,44 @@
+import validator from 'validator';
+
+import {
+  PASSWORD_CONFIRM_ERROR,
+  PASSWORD_EMPTY_ERROR,
+  PASSWORD_EQUAL_ERROR,
+  PASSWORD_WEAK_ERROR,
+} from '../config/messages';
+
+export const strengthValidator = (password) => {
```

Comment:
In general for this file: Since these are validators, you would expect to receive booleans, and maybe throwing errors instead of strings.